packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -170,28 +172,28 @@ function WorklistDataGrid(props) {
       <WorklistCellTag
         key={cellColumnIndex}
         {...tabIndex}
-        className={cx('worklist-data-grid-cell', { masked: cell.isMasked })}
+        className={cx('worklist-data-grid-cell', { masked: cell.isMasked, selectable: !(cell.isMasked || cell.isSelectable === false) })}
         aria-label={cell.isMasked ? intl.formatMessage({ id: 'Terra.worklistDataGrid.maskedCell' }) : undefined}
       >
         <div className={cx('cell-content')}>{cell.content}</div>
       </WorklistCellTag>
     );
   };
 
-  const buildColumn = (columnData) => {
-    const width = columnData.width || props.columnWidth;
+  const buildColumn = (column) => {
+    const width = column.width || props.columnWidth;
     const height = props.columnHeaderHeight;
     return (
       /* eslint-disable react/forbid-dom-props */
-      <th key={columnData.id} className={cx('worklist-data-grid-column-header')} tabIndex="-1" style={{ width, height }}>{columnData.displayName}</th>
+      <th key={column.id} className={cx('worklist-data-grid-column-header', { selectable: !(column.isSelectable === false) })} tabIndex="-1" style={{ width, height }}>{column.displayName}</th>
```

Comment:
nit: can we simplify this to just `selectable: column.isSelectable`

Reply:
@artpark - We cannot use this simple case because we want an undefined value to evaluate to true as well.